### PR TITLE
Add TotalPrice in response of hotels/search endpoint

### PR DESCRIPTION
**Pull request by FelipeBrizola**

### CODE REVIEW

File: src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown [hunk 1]
```diff
@@ -103,6 +103,7 @@ Hotel property object returned by hotel search.
 `position`|[`Geolocation`](#schemageolocation)|-|Geolocation of property.|
 `address`|[`Address`](#schemaaddress)|-|**Required** Address of property.|
 `leadRate`|[`LeadRate`](#schemaleadrate)|-|**Required** The lowest nightly rate averaged over the stay. Not required if `availabilityStatus` is `CLOSED_OUT` or `UNKNOWN`.|
+`totalPrice`|[`TotalPrice`](#schematotalpricehotelproperty)|-|**Required** Details about the total pricing associated with stay.
```
Comment (kunalshah01) [inline src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown:106]: Anything filled by supplier - we cannot mark it as required.

File: src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown [hunk 2]
```diff
@@ -461,6 +462,23 @@ The lowest nightly rate averaged over the stay.
...
+
+Details about the total pricing associated with stay.
+
+|Name|Type|Format|Description|
+|---|---|---|---|
+`totalOverStay`|[`TotalOverStay`](#schematotalOverStay)|-|**Required** |
+`isTaxInclusive`|`boolean`|`true` / `false`|Whether or not taxes are included.|
+`isFeesInclusive`|`boolean`|`true` / `false`|Whether or not fees are included.|
+
+## <a id="schematotalOverStay"></a> TotalOverStay
```
Comment (kunalshah01) [inline src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown:475]: we already have #schemaprice with this structure, just use that, no need for new type

File: src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown [hunk 3]
```diff
@@ -461,6 +462,23 @@ The lowest nightly rate averaged over the stay.
 `avgNightlyRate`|[`Price`](#schemaprice)|-|**Required** |
 `isTaxAndFeesInclusive`|`boolean`|`true` / `false`|Whether or not tax and fees are included.|
 
+## <a id="schematotalpricehotelproperty"></a> TotalPrice
+
+Details about the total pricing associated with stay.
+
+|Name|Type|Format|Description|
+|---|---|---|---|
+`totalOverStay`|[`TotalOverStay`](#schematotalOverStay)|-|**Required** |
```
Comment (kunalshah01) [inline src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown:471]:
```suggestion
`totalOverStay`|[`Price`](#schemaPrice)|-|**Required** |
```

File: src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown (hunk 1 above)
Comment (FelipeBrizola) [reply src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown]: Required was removed

**Review by kunalshah01 — COMMENTED** [review]
You also need to update Swagger JSON spec here: https://github.com/SAP-docs/preview.developer.concur.com/blob/main/src/api-explorer/v4-0/HotelService.swagger2.json

File: src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown (hunk 2 above)
Comment (FelipeBrizola) [reply src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown]: schematotalOverStay removed

File: src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown [hunk 4]
```diff
@@ -103,6 +103,7 @@ Hotel property object returned by hotel search.
 `position`|[`Geolocation`](#schemageolocation)|-|Geolocation of property.|
 `address`|[`Address`](#schemaaddress)|-|**Required** Address of property.|
 `leadRate`|[`LeadRate`](#schemaleadrate)|-|**Required** The lowest nightly rate averaged over the stay. Not required if `availabilityStatus` is `CLOSED_OUT` or `UNKNOWN`.|
+`totalPrice`|[`TotalPrice`](#schematotalpricehotelproperty)|-|Details about the total pricing associated with stay.
```
Comment (kunalshah01) [inline src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown:106]:
```suggestion
`totalPrice`|[`TotalPriceHotel`](#schematotalpricehotelproperty)|-|Details about the total pricing associated with stay.
```
use `TotalPriceHotel` to differentiate from `TotalPrice`

**Comment by FelipeBrizola** [comment]
Hi @kunalshah01, thank you for your review.
The requested changes have been implemented, and I’ve reopened the PR for review.
Please let me know if there’s anything else needed.

**Review by kunalshah01 — APPROVED** [review]

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**